include/cert-manager.mk

```diff
@@ -0,0 +1,25 @@
+# https://cert-manager.io/docs/tutorials/acme/nginx-ingress/
+prodserver ?= https:\\/\\/acme-v02.api.letsencrypt.org/directory
+
+.PHONY: cert-manager
+cert-manager: 
+	helm repo add jetstack https://charts.jetstack.io
+	helm repo update
+	helm install cert-manager jetstack/cert-manager \
+	--namespace cert-manager \
+  	--create-namespace \
+  	--version v1.9.1 \
+	--set installCRDs=true
+
+.PHONY: letsencypt-staging
+letsencypt-staging:
+	cat ../include/letsencypt.yaml | sed -E "s/someone@somewhere.io/$(certEmail)/g" | kubectl create -n cert-manager -f -
```

Comment:
Should `someone@somewhere.io` be a configuration variable defined by the user in the `Makefile`?

Reply:
This is the placeholder string being replaced with the variable certEmail from the make file